app/assets/stylesheets/components/beta_tag/_beta_tag.scss

```diff
@@ -0,0 +1,11 @@
+.beta-tag {
+  @include inline-block();
+  @include body(18, 24);
+  font-weight: 700;
+  position: relative;
+  bottom: -$baseline-unit*5;
```

Comment:
Not quite sure we're on the same page but as discussed earlier this
approach is absolutely good to go :)